pkg/test/framework/components/echo/config.go

```diff
@@ -73,7 +73,6 @@ const (
 	UbuntuXenial VMDistro = "UbuntuXenial"
 	UbuntuJammy  VMDistro = "UbuntuJammy"
 	Debian11     VMDistro = "Debian9"
```

Comment:
Not related but this line has a typo Debian11 = Debian9

Reply:
yikes... thanks. Ill send another PR

Reply:
Actually not as bad as I thought, I think its cosmetic. But still good to fix